<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gaussian Blur</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QScrollArea" name="scrollAreaImage">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>431</width>
         <height>489</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollAreaBlur">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>431</width>
         <height>489</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionProcess_Image"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Image"/>
   <addaction name="actionSet_kernel_size"/>
   <addaction name="actionBlur_method"/>
   <addaction name="separator"/>
   <addaction name="actionProcess_Image"/>
   <addaction name="actionGather_stats"/>
   <addaction name="actionWrite_Kernel"/>
   <addaction name="actionWrap"/>
  </widget>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionProcess_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Process Image</string>
   </property>
  </action>
  <action name="actionSet_kernel_size">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kernel size</string>
   </property>
  </action>
  <action name="actionGather_stats">
   <property name="text">
    <string>Gather stats</string>
   </property>
  </action>
  <action name="actionWrite_Kernel">
   <property name="text">
    <string>Write Kernel</string>
   </property>
  </action>
  <action name="actionBlur_method">
   <property name="text">
    <string>blur method</string>
   </property>
  </action>
  <action name="actionWrap">
   <property name="text">
    <string>wrap</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
